src/misc_utils.c

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright(c) 2018-2023 Intel Corporation. All rights reserved.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
```

Comment:
are you Liam?

Reply:
I just moved this function from another file. Git blamed Liam of adding it in this [commit](https://github.com/thesofproject/rimage/commit/b537522a6501f0f7249f26c24eb1a4d8679f4bf1). This commit moves all the code from tools repository, so I'm not sure who is the author. I will remove author entry.